### PR TITLE
docs - tencentcloud resources docs with examples

### PR DESCRIPTION
**Pull request by MaDongdong99**
For the existent resources

### CODE REVIEW

File: tools/c7n_tencentcloud/c7n_tencentcloud/filters/metrics.py [hunk 1]
```diff
@@ -29,7 +29,12 @@
 
 
 class MetricsFilter(Filter):
-    """MetricsFilter"""
+    """Supports metrics filters on resources.
```
Comment (thisisshi) [inline tools/c7n_tencentcloud/c7n_tencentcloud/filters/metrics.py:32]: this should have an actual policy example

**Review by thisisshi — CHANGES_REQUESTED** [review]
one minor comment, otherwise lgtm

File: tools/c7n_tencentcloud/c7n_tencentcloud/filters/metrics.py (hunk 1 above)
Comment (MaDongdong99) [reply tools/c7n_tencentcloud/c7n_tencentcloud/filters/metrics.py:32]: done

**Review by thisisshi — APPROVED** [review]
thanks, lgtm

**Pull request merged**